tests/unit/models/article-test.js

```diff
@@ -3,9 +3,12 @@ import {
   test
 } from 'ember-qunit';
 
+import Article from 'uptothetweet/models/article';
+import Tweet from 'uptothetweet/models/tweet';
```

Comment:
'import' is only available in ES6 (use esnext option).<br>'Tweet' is defined but never used.